tests/engines/transit.spec.ts

```diff
@@ -95,7 +95,10 @@ describe("Transit Vault Client", () => {
                 .then((res) => res.data.batch_results);
             const res = await client
                 .decrypt("test", {
-                    batch_input: result,
+                    batch_input: result.map((r) => ({
+                        ciphertext: r.ciphertext,
+                        context: r.context,
+                    })),
```

Comment:
Maybe add another test case that explicitly tests for empty plaintexts.

Reply:
Good idea :) done